google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterOptions.java

```diff
@@ -57,11 +59,23 @@ public abstract class BulkWriterOptions {
   @Nullable
   public abstract ScheduledExecutorService getExecutor();
 
+  /**
+   * Limit on the total number of mutations in-memory.
+   *
+   * @return The maximum number of operations that will be queued.
+   */
+  @Nullable
+  public abstract Integer getMaxPendingOps();
+
+  abstract int getMaxInFlightOps();
```

Comment:
needs a comment here.
BTW, what is the relationship/difference between pendingOps and InFlightOps